### PR TITLE
Add RBS package

### PR DESCRIPTION
**Pull request by soutaro**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is syntax definition of RBS -- the type definition language for Ruby.

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

File: repository/r.json [hunk 1]
```diff
@@ -461,6 +461,17 @@
 				}
 			]
 		},
+		{
+			"name": "RBS Syntax",
```
Comment (braver) [inline repository/r.json:465]: We try to have language packages simply named after the language, without suffixes like "syntax" (or "highlighting" or "language" etc)

Comment (soutaro) [reply repository/r.json:465]:
```suggestion
			"name": "RBS",
```

File: repository/r.json [hunk 2]
```diff
@@ -461,6 +461,17 @@
 				}
 			]
 		},
+		{
+			"name": "RBS",
+			"details": "https://github.com/soutaro/sublime-rbs-plugin",
+			"labels": ["ruby", "rbs"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
Comment (soutaro) [inline repository/r.json:470]:
```suggestion
					"sublime_text": ">3092",
```

**Comment by soutaro** [comment]
Thanks @braver,

I think I have resolved the warning and your feedback!

**Pull request merged**